src/js/view/captionsrenderer.js

```diff
@@ -237,11 +237,18 @@ const CaptionsRenderer = function (viewModel) {
 
             // Use screen dimensions when in fullscreen on mobile devices
             if (_model.get('fullscreen') && OS.mobile) {
-                // device is in portrait mode when window.orientation = 0 || 180
-                const portraitMode = !(window.orientation % 180);
                 const { screen } = window;
-                containerHeight = portraitMode ? screen.availHeight : screen.availWidth;
-                containerWidth = portraitMode ? screen.availWidth : screen.availHeight;
+                if (screen.orientation) {
+                    containerHeight = screen.availHeight;
+                    containerWidth = screen.availWidth;
+                } else {
+                    // availHeight and availWidth don't change in iOS when the orientation changes
+                    // iOS device is in portrait mode when window.orientation = 0 || 180
+                    const portraitMode = !(window.orientation % 180);
```

Comment:
Did you mean to use both `screen` and `window`?

Reply:
Yep. `window.orientation` is the older standard which will be dropped in the future (see: https://developer.mozilla.org/en-US/docs/Web/API/Window/orientation)

`window.screen.orientation` isn't yet supported on iOS, so we're using `window.orientation` as the fallback.